src/scripts/config/generate_config_file.sh

```diff
@@ -80,6 +81,9 @@ while [ $# -gt 0 ] ; do
     -2 | --tier2-sub-id)
       shift
       mlz_tier2_subid="$1" ;;
+    -3 | --tier3-sub-id
+      shift
+      mlz_tier3_subid="$1" ;;
```

Comment:
`mlz_tier3_subid` looks unused, if it needs to make it to the .mlzconfig file, we can add it with the `append_optional_args` like on L132-135

Reply:
Args added